ecc_enclave/enclave/CMakeLists-common-app-enclave.txt

```diff
@@ -52,15 +52,16 @@ target_link_libraries(enclave "${SGX_COMMON_CFLAGS} -Wl,--no-undefined -nostdlib
 
 add_custom_command(TARGET enclave
     POST_BUILD
+    COMMENT "Create enclave signer's key"
+    COMMAND openssl genrsa -3 -out ${CMAKE_BINARY_DIR}/enclave_signer_private.pem 3072
```

Comment:
why do we create a new signer key for every build?

Reply:
> why do we create a new signer key for every build?

You forgot we are Intel, we want users to use as much cycles as possible  ... ;-)